### DISCUSSION TITLE
Steam steamwebhelper

### DISCUSSION THREAD

**Discussion by TheBeginnerPro** [Whisky-App/Whisky#1198] [Q&A] [answered]
Hi friends, who had the error steamwebhelper not responding? How to solve it?
<img width="687" alt="Знімок екрана 2024-11-06 о 01 55 29" src="https://github.com/user-attachments/assets/bb650cda-ee84-4947-9b9d-2a1b8fa47048">

**Comment by akoasatom** [comment, marked as answer]
https://www.reddit.com/r/macgaming/comments/192xrdm/steamwebhelper_stopped_working_in_whisky/
